packages/react-components/src/components/VideoGallery.tsx

```diff
@@ -259,28 +259,35 @@ export const VideoGallery = (props: VideoGalleryProps): JSX.Element => {
       onCreateLocalStreamView && onCreateLocalStreamView(localVideoViewOptions);
     }
     return (
-      <VideoTile
-        key={localParticipant.userId}
-        userId={localParticipant.userId}
-        renderElement={
-          localVideoStream?.renderElement ? (
-            <>
-              {
-                /* @conditional-compile-remove(call-with-chat-composite) @conditional-compile-remove(local-camera-switcher) */
-                localCameraCycleButton(localVideoCameraCycleButtonProps)
-              }
-              <StreamMedia videoStreamElement={localVideoStream.renderElement} />
-            </>
-          ) : undefined
-        }
-        showLabel={!(shouldFloatLocalVideo && isNarrow)}
-        displayName={isNarrow ? '' : strings.localVideoLabel}
-        initialsName={localParticipant.displayName}
-        styles={localVideoTileStylesThemed}
-        onRenderPlaceholder={onRenderAvatar}
-        isMuted={localParticipant.isMuted}
-        showMuteIndicator={showMuteIndicator}
-      />
+      <Stack
+        tabIndex={0}
+        aria-label={'Tile keyboard movement Dropdown Hidden'}
+        aria-role={'drop down'}
+        aria-live={'polite'}
```

Comment:
Just want to double check this is needed as the documentation implies to try avoid this (well to make sure it's used correctly):
![image](https://user-images.githubusercontent.com/2684369/160171419-594480dc-a264-409a-a545-85027bee49c4.png)

What content is changed and read out inside this that needs the polite property?

Reply:
oh I will remove it, my understanding was that the narration of this control would be read out in standard order when scan mode is active but I will remove it based on this.